content/en/docs/concepts/workloads/controllers/_index.md

```diff
@@ -1,5 +1,61 @@
 ---
 title: "Workload Resources"
 weight: 20
+simple_list: true
 ---
 
+Kubernetes provides several built-in APIs for declarative management of your
+{{< glossary_tooltip text="workloads" term_id="workload" >}}
+and the components of those workloads.
+
+Ultimately, your applications run as containers inside
+{{< glossary_tooltip term_id="Pod" text="Pods" >}}; however, managing individual
+Pods would be a lot of effort. For example, if a Pod fails, you probably want to
+run a new Pod to replace it. Kubernetes can do that for you.
+
+You use Kubernetes API to create a workload
+{{< glossary_tooltip text="object" term_id="object" >}} that represent a higher level
+abstraction than a Pod, and then the Kubernetes
+{{< glossary_tooltip text="control plane" term_id="control-plane" >}} automatically manages
+Pod objects on your behalf, based on the specification for the workload object you defined.
+
+The built-in APIs for managing workloads are:
```

Comment:
I'm unclear why the overview of these workload resources is repeated here.

Reply:
Do you mean, why should https://kubernetes.io/docs/concepts/workloads/controllers/ list the different APIs for managing workloads?

The primary aim of this PR is to improve that page specifically; I think the new text improves on the existing autogenerated overview. Right now, if you don't know Kubernetes much and a website takes you to https://kubernetes.io/docs/concepts/workloads/controllers/, it's not obvious what you should read or why.

Realistically, https://kubernetes.io/docs/concepts/workloads/controllers/ is so basic that people won't hyperlink to it. We should fix that.

Reply:
Providing overview of workload resources is totally okay to me. What confused me is that the same text now appear in two places. Maybe we can drop the text in the controllers index page?

Reply:
We could. I kept it because someone might already have hyperlinked to that other page.

Reply:
(Although we could edit https://kubernetes.io/docs/concepts/workloads/ to not mention terms like Deployment, ReplicaSet, StatefulSet etc, people might link there on the assumption that we do mention those things directly.)

Reply:
I read through the main pages and I agree that the content appears to be somewhat shared between the pages. The Workloads section contains the Pods and Workload Resources folders.
The Workloads index page defines a workload, provide descriptions of the workload resources, and list related tasks.
The Workload Resources index page defines the same workload resources.

Generally, I don't think you need the paragraphs about the workload resources in the Workloads index page. You can assume that readers will move into this folder to read about the workload resources. Any information that is missing from the Workload Resources index page can be moved from the index page.

Does that make sense?

Reply:
> You can assume that readers will move into this folder to read about the workload resources.

I agree that we **can** assume that. However, the reason for this is something I am (slightly) ashamed about.
I still think that _https://kubernetes.io/docs/concepts/workloads/controllers/ is so basic that people won't hyperlink to it_ - and that we should fix that. I want people writing other web pages to see https://kubernetes.io/docs/concepts/workloads/controllers/ as something that's useful to link to or to mention to people on Slack, etc.

Reply:
> Realistically, https://kubernetes.io/docs/concepts/workloads/controllers/ is so basic that people won't hyperlink to it.

I'm (slightly) ashamed of this; that's why I want to see it fixed.